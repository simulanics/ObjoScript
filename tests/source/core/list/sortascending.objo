var list = [2,5,1,4,3]
var list2 = ["dog", "house", "airplane", "cat", "goat", "elephant", "bat", "frog"]
list.sortAscending()
list2.sortAscending()

System.print(list)
System.print(list2)