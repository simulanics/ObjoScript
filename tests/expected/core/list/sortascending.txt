[1, 2, 3, 4, 5]
[airplane, bat, cat, dog, elephant, frog, goat, house]